src/routes/memberLogin.jsx

```diff
@@ -1,16 +1,50 @@
 import Navbar from 'Components/navbar';
-import { useId, useRef } from 'react';
+import { useId, useRef, useEffect } from 'react';
 import { useNavigate } from "react-router-dom";
+import { useCookies } from 'react-cookie';
 
 
+// Sample user database
+const loginInfo = [
+    {username: "musician", password: "1234"},
+    {username: "creator", password: "1234"}
+]
+
+const hoursToSeconds = (hours) => hours * 3600;
+
 export default function MemberLogin() {
     const usernameId = useId(), passwordId = useId();
     const username = useRef(), password = useRef();
     const navigate = useNavigate();
 
+    // https://github.com/reactivestack/cookies/tree/master/packages/react-cookie/#getting-started
+    const [cookies, setCookie] = useCookies(['username']); //username cookie, used for login
+
+   useEffect(() => { 
+        if (cookies.username != (null || "undefined")) //user is logged in
```

Comment:
this does not work. (null || "undefined") evaluates to true since "undefined" is a non-empty string.
we must allow unsigned users to access this page